content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1280,9 +1283,11 @@ Defines a multi-field ID (composite ID) on the model.
 
 #### Arguments
 
-| Name     | Required | Type               | Description                                                      |
-| -------- | -------- | ------------------ | ---------------------------------------------------------------- |
-| `fields` | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]` |
+| Name     | Required | Type               | Description                                                                 |
+| -------- | -------- | ------------------ | --------------------------------------------------------------------------- |
+| `fields` | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]`            |
+| `name`   | No       | `String`           | The name of the field in the generated client - defaults to `field1_field2` |
```

Comment:
Is `name` new for `@@id` @do4gr?

Reply:
"@@id(map: database_pk_name) ← gets name = expose to Client (if missing = still constructed from field names)"